### PR TITLE
Qt patches

### PR DESCRIPTION
**Pull request by bdbcat**
Implement glCanvas attributes, removing hard-coded values.

### CODE REVIEW

File: include/wx/qt/glcanvas.h [hunk 1]
```diff
@@ -87,4 +87,7 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasBase
     wxDECLARE_CLASS(wxGLCanvas);
 };
```
Comment (transmitterdan) [inline include/wx/qt/glcanvas.h:89]: @bdbcat  There is a request to move any includes not required by wxWidgets users out of the wx header files and down into the implementation file.  Are these 2 includes needed by end users?

**Review by transmitterdan — CHANGES_REQUESTED** [review]
@bdbcat Why did this PR drag in the changes to expat and tiff submodules.  I don't think we want to do that.  Can you resumit PR without including submodule changes?

File: include/wx/qt/glcanvas.h (hunk 1 above)
Comment (bdbcat) [reply include/wx/qt/glcanvas.h:89]: Dan...
Need more context.  What two include files, what source file?

**Comment by bdbcat** [comment]
Dan...
I will resubmit.
I think I have figured out the last issue with DTOR on wxWindow.  I will incorporate that patch, as well.

File: include/wx/qt/glcanvas.h (hunk 1 above)
Comment (transmitterdan) [reply include/wx/qt/glcanvas.h:89]: There are a bunch of files in the PR coming in from the submodule libraries (expat, pnt, tiff).  Look at the file compare at github.com (can't see it in git diff probably).

File: include/wx/qt/glcanvas.h [hunk 2]
```diff
@@ -87,4 +87,7 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasBase
...
 
+#include <QtWidgets/QGestureRecognizer>
```
Comment (transmitterdan) [inline include/wx/qt/glcanvas.h:90]: @bdbcat if these include statements don't need to be exposed in the public header they should be moved into the .cpp file that needs them.

**Pull request closed without merging**